scripts/run-eslint.ts

```diff
@@ -0,0 +1,405 @@
+import chalk from 'chalk';
```

Comment:
This script is copied from the core Eslint fix PR https://github.com/MetaMask/core/pull/5132/files